tests/test_connection.py

```diff
@@ -3541,6 +3541,9 @@ def test_upgrade_statickey_onchaind(node_factory, executor, bitcoind):
     l1.daemon.wait_for_log('option_static_remotekey enabled at 1/1')
 
     # Pre-statickey penalty works.
+    # FIXME: Without this sleep, l1 will broadcasts one tx more compared to good
+    # case, causing `wait_for_onchaind_broadcast` to timeout.
+    time.sleep(5)
```

Comment:
Thanks, I reproduced this issue, and have the proper fix I think... https://github.com/ElementsProject/lightning/pull/4941